### PR TITLE
Fixes Numero Three

### LINKED ISSUE #9352: MedalBoxes stuck
MedalBoxes are all buggy and blocked.

### PR DESCRIPTION
**Pull request by AtnerNT**
Fixes #9352
### Changelog:
CL: [Fix] Updated supermatter to new telecomms, Supermatter alerts now once again appear in chat.
CL: [Fix] Fixed foam visuals not appearing for clients.
CL: [Fix] Fixed Cloning pod and DNA scanner to no longer be pushable by players and wind.
CL: [Fix] Set the clearance of medal boxes to their appropriate heads instead of NULL

### CODE REVIEW

**Comment by MaxIsJoe** [comment]
dats a lot of fixes, damn. great job Atner

File: UnityProject/Assets/Scripts/Objects/Engineering/SuperMatter.cs [hunk 1]
```diff
@@ -1065,16 +1069,45 @@ private IEnumerable<Collider2D> GetNearbyEntities(Vector3 centrepoint, int mask,
...
+
+			chatEvent.channels = ChatChannel.Engineering;
+			
+			InfluenceChat(chatEvent);
 
 			if (sendToCommon)
 			{
-				Chat.AddCommMsgByMachineToChat(gameObject, message, ChatChannel.Common,
-					Loudness.SCREAMING, broadcasterName: "Supermatter Warning System: ", language: LanguageManager.Common);
+				chatEvent.channels = ChatChannel.Common;
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Objects/Engineering/SuperMatter.cs]:
```suggestion
				chatEvent.channels |= ChatChannel.Common;
```

Bitwise OR so it goes to both common and engineering

**Comment by corp-0** [comment]
would you mind being a bit more verbose in the changelog? Also use past tense. Remember those lines are to be read in the changelog page and ingame changelog by players.

**Comment by AtnerNT** [comment]
> would you mind being a bit more verbose in the changelog? Also use past tense. Remember those lines are to be read in the changelog page and ingame changelog by players.

That better?

**Review by ThatDan123 — APPROVED** [review]

**Pull request merged**